utils/oscap_docker_python/oscap_docker_util.py

```diff
@@ -120,16 +120,19 @@ def oscap_chroot(self, target, image, chroot_path, *oscap_args):
                    "PRIMARY_HOST_NAME"] = "{0}-{1}".format(target, image)
         cmd = ['oscap'] + [x for x in oscap_args]
         try:
-            run = subprocess.check_output(cmd)
-        except Exception as error:
-            print("\nCommand: {0} failed!\n".format(" ".join(cmd)))
-            print("Error was:\n")
-            print(error)
-
-            # Clean up
-            self._cleanup_by_path(chroot_path)
-
-            sys.exit(1)
+            run = subprocess.check_output(cmd, stderr=subprocess.STDOUT)
```

Comment:
`Subprocess.call` doesn't allow to check the output of command, which we need.

From https://pymotw.com/2/subprocess/:
> The standard input and output channels for the process started by call() are bound to the parent’s input and output. That means the calling programm cannot capture the output of the command. 

But we can do it using `subprocess.Popen()` and `Popen().communicate`